nwplus/livesite
# Part 1 of the hacker app has been hooked up

### I've hooked up part 1 of the application to firebase.
Check out BasicInfo.js and Part1.js for examples on how to set this up. *You don't need to do anything to enable autosave besides calling useHackerApplication*

### fixed some bugs in hacker app context
- Hacker app now properly creates new entry (instead of overwriting template)
- Ethnicity map added to hackertemplate
- hacker app loads until application is ready
- I added a new container over the navbar for part1-3 of the application so that it will save if the user logs out.

### HOW TO TEST
1. Click on the preview URL
2. Login
3. go to `/application/part-1` (since you will be redirected to `/judging`
4. Fill out some or all of the formm
5. Click logout
6. repeat 2 and 3

You should see your changes saved!
